demos/TicTacToe/game_resources/Classes/main_menu_scene.h

```diff
@@ -30,6 +31,7 @@ using cocos2d::Size;
 using cocos2d::Sprite;
```

Comment:
I probably should have caught this in an earlier review, but avoid putting using statements in headers. Putting these here causes them to leak into every cc and header file that includes this header, which is not great.

Reply:
BTW I'm totally fine with you making this change in a follow up PR if it would pollute this one too much with unrelated changes.

Reply:
Sounds good I will create a story for this.